linode/objects/nodebalancer/config.py

```diff
@@ -59,11 +59,11 @@ def create_node(self, label, address, **kwargs):
         result = self._client.post("{}/nodes".format(NodeBalancerConfig.api_endpoint), model=self, data=params)
         self.invalidate()
 
-        if not 'id' in result:
+        if not 'id' in result.get('data')[0]:
             raise UnexpectedResponseError('Unexpected response creating node!', json=result)
 
         # this is three levels deep, so we need a special constructor
-        n = NodeBalancerNode(self._client, result['id'], self.id, self.nodebalancer_id, result)
+        n = NodeBalancerNode(self._client, result.get('data')[0]['id'], self.id, self.nodebalancer_id, result)
```

Comment:
this isn't right - `POST /nodebalancers/:id/configs:/id/nodes` returns a raw object with no envelope